ee/fleetctl/updates.go

```diff
@@ -87,6 +99,14 @@ func updatesInitCommand() *cli.Command {
 	}
 }
 
+func mustParseDuration(s string) time.Duration {
+	d, err := time.ParseDuration(s)
+	if err != nil {
+		panic(err)
```

Comment:
this seems safe as long as the duration variables don't change in the future, but guessing it's a panic instead of a returned err to ensure we catch this in tests?

Reply:
Correct. This should panic right away when executing `fleetctl updates` commands.